docs/content/guides/operator.mdx

```diff
@@ -1,28 +1,33 @@
 ---
 title: Operator Guides
-description: Guides for operators on the Sui network. Whether you are running a Full node for your dApp or operating as a validator on the Sui network, these guides help you setup your environment and operate your network.
+description: Guides for operators on the Sui network. Whether you are running a Full node for your dApp or operating as a validator on the Sui network, these guides help you set up your environment and operate your network.
 ---
 
-Operator guides demonstrate how to run Full nodes on Sui, whether as a validator or Full node operator. 
+Operator guides demonstrate how to run Full nodes on Sui, whether as a validator or operator of a Full node to support your dApp, as well as how to integrate SUI into an exchange. 
 
 ## Sui Full node operators
```

Comment:
^ Ditto - capitalization

Reply:
This one is also correct as is. The "correct" way to write Full node, is Full node. And because the heading is not the title of a page, it's sentence case. I'm not a fan of its current form, either, for what it's worth.